### PR TITLE
Update Readme.md broken links, Added other documentation files

### LINKED ISSUE #2: Broken link for Contribution guide, change log and troubleshooting guide
**Describe the bug**
The Readme link for the above are all broken

### PR DESCRIPTION
**Pull request by chaitak-gorai**
## Description
Documentation files like CONTRIBUTING.md and CODE_OF_CONDUCT.md are important for any open-source project. They provide guidance to the contributors and maintainers on how to contribute and behave in the community and ensure that the project is welcoming to new contributors and sets clear expectations for everyone involved.

## Changes Made
- Added `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` files
- Updated broken links in `README.md`
- Added `CHANGELOG.md` generated by `generate_changelog.py` script and called as a `pre-push` git hook.

## Issue Resolved
- Resolves #2

Maintainers @muarachmann  please review my PR and suggest me some changes if needed.
Thanks for considering this pull request. 😊😊

### CODE REVIEW

**Comment by muarachmann** [comment]
thanks will take a look at this. Also do not wait on my approval before moving to the next tasks. You can submit as many PR as possible

**Comment by chaitak-gorai** [comment]
> thanks will take a look at this. Also do not wait on my approval before moving to the next tasks. You can submit as many PR as possible

Sure and thanks :)

**Comment by muarachmann** [comment]
Hi @chaitak-gorai this is cool but for now we won't want to generate the change logs as thus. We can manually have the change logs or consider this in the future. Can you adjust that?

**Comment by chaitak-gorai** [comment]
> Hi @chaitak-gorai this is cool but for now we won't want to generate the change logs as thus. We can manually have the change logs or consider this in the future. Can you adjust that?

Sure, I will do the necessary changes.

**Comment by chaitak-gorai** [comment]
## Updated 
Based on the changes you asked for, I have removed the CHANGELOG and fixed the links in the README.
## Changes Made
- Removed `CHANGELOG.md` , `generate_changelog.py`  and `pre-push` git hook.
- Updated  links in `README.md`

@muarachmann please have a look. Please let me know if any more changes are needed.
Thank You.

File: README.md [hunk 1]
```diff
@@ -26,12 +26,11 @@ We are collaborating closely with the [LibreHealth Project](http://librehealth.i
...
 3. [Usage](#usage)
 4. [Documentation](#documentation)
-5. [Change log](#change-log)
-6. [Testing](#testing)
-7. [Contributing](#contributing)
-8. [Security](#security)
-9. [Credits](#credits)
-10. [License](#license)
+5. [Testing](#testing)
+6. [Contributing](#contributing)
```
Comment (muarachmann) [inline README.md]: Nice @chaitak-gorai , can this readme involve the tech stack we are using? at least for a start

Comment (chaitak-gorai) [reply README.md]: Sure @muarachmann ,
Shall I pick up the tech stacks mentioned in the about section of the repository?

**Comment by muarachmann** [comment]
Yes and links to them is necessary

**Comment by chaitak-gorai** [comment]
Sure @muarachmann ,
I have added the tech stack section in the README.md and also linked the official website or docs of these.

**Comment by muarachmann** [comment]
LGTM We can always improve this. Nice work @chaitak-gorai

**Pull request merged**